src/Model/common.hpp

```diff
@@ -44,112 +44,118 @@
 
 #include "Initializer/typedefs.hpp"
 #include "generated_code/init.h"
+#include "Geometry/MeshTools.h"
+#include "Numerical_aux/Transformation.h"
+
+#include "Model/common_datastructures.hpp"
+#include "Equations/elastic/Model/datastructures.hpp"
+#include "Equations/anisotropic/Model/datastructures.hpp"
+#include "Equations/viscoelastic2/Model/datastructures.hpp"
 
 namespace seissol {
   namespace model {
-    using Matrix99 = Eigen::Matrix<real, 9, 9>;
+    using Matrix99 = Eigen::Matrix<double, 9, 9>;
```

Comment:
Please don't introduce aliases in header files -> they are defined in places you wouldnt expect